Modules/SQLDBServer/2.0/Tests/parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "administratorLogin": {
+            "value": "contoso"
+        },
+        "administratorLoginPassword": {
+            "value": "password"
+        },
+        "serverName": {
+            "value": "foo"
+        },
+        "diagnosticLogsRetentionInDays": {
+            "value": 365
+        },
+        "diagnosticStorageAccountId": {
+            "value": "foo"
```

Comment:
Should this be based on Contoso or similar standard naming convention ?

Reply:
These are dummy values, technically it can be whatever